RNZoomUs.podspec

```diff
@@ -10,9 +10,8 @@ Pod::Spec.new do |s|
                    DESC
   s.homepage     = "https://github.com/mieszko4/react-native-zoom-us"
   s.license      = "MIT"
-  # s.license      = { :type => "MIT", :file => "FILE_LICENSE" }
   s.author             = { "author" => "author@domain.cn" }
-  s.platform     = :ios, "8.0"
+  s.platform     = :ios, "9.0"
```

Comment:
that should not be breaking change, right?

Reply:
No. Libraries moves this number forward.
9 is good enough. Many libraries already have v10-11.

Breaking change that zoom uses some ios internal frameworks now.
 It is managed in ZoomSDK Pod already.
 So far looks ok.